pipeline/metadata/flatten.py

```diff
@@ -531,10 +542,14 @@ def _process_satellite_v2_responses(
     """
     responses = scan.get('response', [])
     if responses:
+      is_domain_control = _is_control_url(scan['test_url'])
```

Comment:
I changed this to false and added a comment

Reply:
I can't comment on line 571 but it might make sense to eventually set `row['is_control']` in the responses loop. Only the test domain rows are being yielded there so it doesn't make a difference for this PR